### ISSUE TITLE
Highlight matches does not work with color codes

### ISSUE THREAD

**Issue by urbanware-org** [urbanware-org/salomon#17]
When using color codes to colorize lines, the `--highlight-matches` argument does not have any effect.

For example, when executing the command

```
./salomon.sh -a analyze -c log_colors.cfg -i samples/foobar.log -f "2014" --highlight-matches
```

while the color config file has the following instruction

```
214         warn
```

to colorize lines containing the term `warn` using color code `214`, the filter term is not being highlighted. See below.

![no-match-highlight](https://user-images.githubusercontent.com/19925115/48441322-81dcc600-e78b-11e8-8645-e72d91b57aba.png)

**Comment by urbanware-org** [comment]
So far, the matches are now highlighted, but their text is not displayed.

**Comment by urbanware-org** [comment]
**Fixed** in version **1.11.1**.
